src/components/Teacher/TeacherDashBoard.jsx

```diff
@@ -0,0 +1,30 @@
+import React,{useEffect}from "react";
+import styles from "./teacher.module.css";
+import Students from "../Students";
+import { useAtom } from "jotai";
+import { ClassAtom, StudentAtom, StudentOfClass } from "../jotai";
+
+export default function TeacherDashBoard() {
+  const [class1, setclass1] = useAtom(ClassAtom);
+  const [studentOfClass, setStudentOfClass] = useAtom(StudentOfClass);
+  const [students, setStudents] = useAtom(StudentAtom);
+
+  useEffect(() => {
+    
+    setStudentOfClass( students.filter(student => student.class === class1));
+  
+  }, [class1]);
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.navbar}>
+        <button className={styles.class} onClick={()=>setclass1('8th')}>8th class</button>
+        <button className={styles.class} onClick={()=>setclass1('9th')}>9th class</button>
+        <button className={styles.class} onClick={()=>setclass1('10th')}>10th class</button>
```

Comment:
class1 should be a meaningful name and values should not be some random strings. It would be best to define a const to hold all the classes codes.